examples/js/controls/DragControls.js

```diff
@@ -21,11 +21,15 @@ THREE.DragControls = function ( _objects, _camera, _domElement ) {
 	var _intersection = new THREE.Vector3();
 
 	var _selected = null, _hovered = null;
-
+	var _planeNormal = null;
 	//
 
 	var scope = this;
 
+	this.setPlaneNormal = function ( planeNormal ) {
+		_planeNormal = planeNormal;
```

Comment:
I would implement it like so:
```javascript
this.setPlaneNormal = function ( normal ) {

	_planeNormal.copy( normal );

};
```
You will have to modify the remainder of the changes, however.

Reply:
Thanks for the suggestion, still new to open source! Would you recommend amending or making a new commit?

Reply:
Not necessarily. You can modify this one if you want. It's up to you.